curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa3589eacea3f48c6300ae.md

```diff
@@ -56,9 +56,9 @@ assert(
 ```html
 <html>
   <body>
-    <h1>CatPhotoApp</h1>
     <main>
       <section>
+        <h1>CatPhotoApp</h1>
```

Comment:
As far as accessibility is concerned, it doesn't matter if the `h1` is in the `section` or before it. The sections in this project are not providing any accessibility related functionality since they don't have accessible names (they are basically just the same as divs). But semantically, I would tend to agree that the `h1` should be outside of the `section` since it is the heading for the entire document, not just that section.

Reply:
This is a good point. I made the changes for this. Should be good now. 

Also changed so this won't close the issue and I'll submit another PR hopefully soon to remove the header from the other challenge.